docs/cpp/config-clang-mac.md

```diff
@@ -229,6 +229,8 @@ By default, the C++ extension won't add any breakpoints to your source code and
 
 Change the `stopAtEntry` value to `true` to cause the debugger to stop on the `main` method when you start debugging.
 
+Also make sure the value of `preLaunchTask` matches the `label` of the build task in task.json file.
```

Comment:
Thank you for adding this.

We could also make it more direct, e.g.:

Ensure that the `preLaunchTask` value matches the `label` of the build task in the task.json file.